### DISCUSSION TITLE
Position when using `showNotification`

### DISCUSSION THREAD

**Discussion by Satont** [mantinedev/mantine#3247] [Feature requests]
It might be good to able set position when using `showNotification` function from `@mantine/modals` package

**Comment by rtivital** [comment]
You can set notifications positions in NotificationsProvider. I do not see any reason why you would need to display notifications at different positions.

**Reply by Satont** [reply]
My default position is center, and alerts overlap with modal.

Mainly i use drawer a lot, thats why i cannot use `right`. 

Left section is free, but i not feel it's good ux.

So solution might be have pos property for alert, or center modal. 

Anyways it's goos feature to have, right?

**Reply by lardissone** [reply]
A real use case for this is: I have global toasts/notifications for confirmation and alerts (form submissions feedback, etc) that I display as a top-centered notification with the message, and then I have realtime notifications from the server using websockets, that I would like to keep them in the right side, since it's pretty much a standard for notifications.

**Comment by dimosmera** [comment]
This is now supported in 7.12.0 https://github.com/mantinedev/mantine/releases/tag/7.12.0
